lib/blurb/report_requests.rb

```diff
@@ -24,6 +25,7 @@ def create(
       }
       payload[:segment] = segment if segment
       payload[:tactic] = SD_TACTIC if @campaign_type.to_sym == :sd
+      payload[:creative_type] = creative_type if @creative_type
```

Comment:
@Sophia-bhs I think this needs to be `creative_type` rather than `@creative_type`